Automattic/newspack-plugin
# fix: remove unneeded blocks from patterns

### All Submissions:


### Changes proposed in this Pull Request:

This PR matches https://github.com/Automattic/newspack-blocks/pull/1377 and removes unnecessary group and columns blocks from the Homepage patterns in the site setup wizard. 

* The group blocks that have been removed don't have background colours.
* The columns that have been removed are cases where they could be replaced with the Homepage Posts block set in 'grid' layout mode.

See 1200550061930446-as-1204083986338479

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Run through the set-up wizard 12 times (sorry!); and pick a different homepage starter pattern each time.
3. Confirm there are no errors in the editor.
4. Review the code changes and confirm nothing unexpected is getting removed.

### Other information:
